modular_skyrat/master_files/code/datums/traits/neutral.dm

```diff
@@ -13,9 +13,9 @@
 
 /datum/quirk/personalspace
 	name = "Personal Space"
-	desc = "You'd rather people keep their hands to themselves, and you won't let anyone touch your ass.."
-	gain_text = span_notice("You'd like it if people kept their hands off your ass.")
-	lose_text = span_notice("You're less concerned about people touching your ass.")
+	desc = "You'd rather people keep their hands to themselves."
+	gain_text = span_notice("You'd like it if people kept their hands off your privates.")
```

Comment:
Butt feels more appropriate than privates, frankly.

Reply:
It does, yes. More SS13 like, thank you.

Reply:
Actually. What about rear end?

Reply:
Eh, not so sure. Butt feels more to the point.

Reply:
Very well.